test/test-errortracker.js

```diff
@@ -364,7 +364,7 @@ describe('Test how server responds to requests', function() {
 
   it('Should unminify Stacktraces', function() {
     const stackTrace = ` at https://examplet.com/www/js/min.js:2:28
-      at s (https://example.com/www/js/min.js:2:28)
+      at n (https://example.com/www/js/min.js:2:28)
       at https://examples.com/www/js/min.js:2:28
```

Comment:
It was not in the sourcemap. You randomly added the S yesterday to test your theory.